### PR TITLE
Edgesite documentation

### PR DESCRIPTION
**Pull request by naveensriram**
**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:
Edgesite documentation

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by sids-b** [comment]
images/edgesite/EdgeSite_arch.PNG is added .... but I think images/EdgeSite_arch.PNG should be in deleted files as I think this file has moved locations

File: edgesite/README.md [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+
+EdgeSite helps running lightweight clusters at edge.
+
+## Motivation
+
+There are scenarios user need to run a standalone Kubernetes cluster at edge to get full control and improve the offline scheduling capability. There are two scenarios user need to do that:
+
+* The edge cluster is in CDN instead of the user's site
+
+  The CDN sites usually be large around the world and the network connectivity and quality cannot be guaranteed. Another factor is that the application deployed in CDN edge do not need to interact with center usually. For those deploy edge cluster in CDN resources, they need to make sure the cluster is workable without the connection with central cloud not only for the deployed applicatons but also the schedule capabilities. So that the CDN edge is manageable regardless the connection to one center.
```
Comment (rohitsardesai83) [inline edgesite/README.md]: The CDN sites usually be large  --> The CDN sites are usually large

Comment (rohitsardesai83) [reply edgesite/README.md]: For those deploy edge cluster   --> For those who deploy edge cluster

Comment (rohitsardesai83) [reply edgesite/README.md]: but also the schedule capabilities.  --> this doesn't sound correct. can you please make it more clear.

**Comment by rohitsardesai83** [comment]
we need to add a placeholder for edgesite in the https://github.com/kubeedge/kubeedge/blob/master/docs/index.rst as well ?

**Comment by rohitsardesai83** [comment]
let's continue to improve the doc.

**Comment by rohitsardesai83** [comment]
we can merge once we have updated the document with the release tars links.

**Comment by sids-b** [comment]
I think it's ok to not have release tar link in this doc as we may ned to keep changing this doc too frequently to maintain the link

**Pull request merged**